tests/JMS/Serializer/Tests/Serializer/XmlSerializationTest.php

```diff
@@ -89,7 +89,7 @@ public function testPropertyIsCollectionOfObjectsWithAttributeAndValue()
 
     /**
      * @expectedException JMS\Serializer\Exception\InvalidArgumentException
-     * @expectedExceptionMessage The document type "<!DOCTYPE author [<!ENTITY foo SYSTEM "php://filter/read=convert.base64-encode/resource=XmlSerializationTest.php">]>" is not allowed. If it is safe, you may add it to the whitelist configuration.
+     * @expectedExceptionMessage The document type "<!ENTITY foo SYSTEM "php://filter/read=convert.base64-encode/resource=XmlSerializationTest.php">" is not allowed. If it is safe, you may add it to the whitelist configuration.
```

Comment:
That's a good question.
On my computer (and before the modification) this test was failing in all versions of php except for 5.5
I then noticed that the behavior was inconsistent depending on the version of php. I did a little search and i found that i was hitting a PHP bug (default php version in ubuntu 14.04 is 5.5.9  https://bugs.php.net/bug.php?id=67081). After updating, they were all failing so i assumed the one who wrote the test was also hitting the bug.